src/NavBar/NavBar.jsx

```diff
@@ -21,6 +21,7 @@ export default function NavBar() {
           <Nav>
             <Nav.Link href="/">Home</Nav.Link>
             <Nav.Link href="add-recipe">Add Recipe</Nav.Link>
+            <Nav.Link href="likes">Likes</Nav.Link>
```

Comment:
I feel like 'liked recipes' would be more clear to the user.

Reply:
Agreed with Megan (doesn't happen often)